### PR TITLE
Add send-amount-row to Storybook

### PR DESCRIPTION
**Pull request by EtDu**

### CODE REVIEW

File: ui/pages/send/send-content/send-amount-row/send-amount-row.stories.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+export default {
+  title: 'SendAmountRow',
+  decorators: [(story) => <Provider store={store}>{story()}</Provider>],
+};
+
+export const SendAmountComponent = () => {
+  const { metamask } = store.getState();
+  const { tokens } = metamask;
+
+  console.log('tokens', store.getState());
```
Comment (darkwing) [inline ui/pages/send/send-content/send-amount-row/send-amount-row.stories.js:22]: Nit: Remove `console.log`

**Review by darkwing — CHANGES_REQUESTED** [review]

File: ui/pages/send/send-content/send-amount-row/send-amount-row.stories.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      decimals: tokens[0].decimals,
+      symbol: tokens[0].symbol,
+    },
+  };
+  return (
+    <SendAmountRow
+      amount={text('Amount', '100000000000000')}
+      inError={boolean('In Error', false)}
+      asset={asset}
+    />
```
Comment (adonesky1) [inline ui/pages/send/send-content/send-amount-row/send-amount-row.stories.js:36]:
```suggestion
 <div style={{ maxWidth: '500px' }}>
      <SendAmountRow
        amount={text('Amount', '100000000000000')}
        inError={boolean('In Error', false)}
        asset={asset}
      />
    </div>
```

Without somehow containing this I'm seeing this:
Screen Shot 2021-09-28 at 11 19 15 AM

**Comment by EtDu** [comment]
@adonesky1 Suggestion applied. Will provide boxes for future components

**Review by darkwing — DISMISSED** [review]

**Comment by adonesky1** [comment]
@EtDu looks like this might need a little rebase/lint care 😄

**Comment by adonesky1** [comment]
@EtDu possibly not but do we want to hook up the MaxButton to do something in Storybook?

File: ui/pages/send/send-content/send-amount-row/send-amount-row.stories.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      symbol: tokens[0].symbol,
+    },
+  };
+
+  return (
+    <div style={{ maxWidth: '500px' }}>
+      <SendAmountRow
+        amount={text('Amount', '100000000000000')}
+        inError={boolean('In Error', false)}
+        asset={asset}
```
Comment (adonesky1) [inline ui/pages/send/send-content/send-amount-row/send-amount-row.stories.js:37]: maybe we could add 
```
updateSendAmount={action('Update Send Amount')}
```
?

**Pull request closed without merging**